content/docs/dvclive/api-reference/live/make_report.md

```diff
@@ -0,0 +1,31 @@
+# Live.make_report()
+
+Generates an HTML Report from the logged data.
+
+```py
+def make_report()
+```
+
+#### Usage:
```

Comment:
Hmmm I just realized all these Usage headers are H4. Shouldn't they be H2 or H3 maybe? Minor

Reply:
I copied the format from the DVC Python API reference (https://github.com/iterative/dvc.org/blob/master/content/docs/api-reference/get_url.md)

Reply:
Hmmm yeah they should all be H3 at most I think. I'll extract this... ⌛